### PR TITLE
docs: update rpm package to 2.2

### PR DESCRIPTION
**Pull request by moonming**
### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -205,7 +205,7 @@ There are several ways to install the Apache Release version of APISIX:
    - install APISIX：
 
    ```shell
-   $ sudo yum install -y https://github.com/apache/apisix/releases/download/2.1/apisix-2.1-0.el7.noarch.rpm
+   $ sudo yum install -y https://github.com/apache/apisix/releases/download/2.2/apisix-2.2-0.x86_64.rpm
```
Comment (spacewander) [inline README.md:208]: Also need to update `how-to-build.md`?

Comment (moonming) [reply README.md:208]: good catch. fixed

File: doc/how-to-build.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ make deps
 ### Installation via RPM package (CentOS 7)
 
 ```shell
-sudo yum install -y https://github.com/apache/apisix/releases/download/2.2/apisix-2.2-0.el7.noarch.rpm
+sudo yum install -y https://github.com/apache/apisix/releases/download/2.2/apisix-2.2-0.x86_64.rpm
```
Comment (spacewander) [inline doc/how-to-build.md:52]: Also need to change its Chinese version.

Comment (moonming) [reply doc/how-to-build.md:52]: got it

**Review by spacewander — APPROVED** [review]

**Review by tokers — APPROVED** [review]

**Pull request merged**